DB/DB_SCHEMA.sql

```diff
@@ -0,0 +1,28 @@
+create type category as enum('PHONE','LAPTOP','DESKTOP','AUDIO','TABLET','EARPHONE');
+
+CREATE TABLE PRDT_INFO(
+    PRDT_INFO_ID SERIAL PRIMARY KEY,
+    PRDT_NAME VARCHAR(255) NOT NULL UNIQUE,
+    PRDT_DESC TEXT,
+    PRDT_TYPE category ,
+    CREATE_DATE TIMESTAMP,
+    UPDATE_DATE TIMESTAMP
```

Comment:
Using the past tense would better align with the semantics, like `created` and `updated`.